composer.json

```diff
@@ -41,7 +42,7 @@
         "symfony/flex": "^1.6",
         "symfony/form": "^4.4",
         "symfony/framework-bundle": "^4.4",
-        "symfony/messenger": "^4.4",
+        "symfony/messenger": "4.4.10",
```

Comment:
Is the delimiter necessary?

Reply:
It's not. Once this goes it'll be fixed. https://github.com/ergonode/backend/pull/771